naas_python/utils/domains_base/authorization.py

```diff
@@ -258,9 +282,11 @@ def _request_token(self):
 
             _environment_mode = os.getenv("NAAS_ENVIRONMENT_MODE", "production")
             if _environment_mode == "production":
+                print(f'🌐 Opening {self._login_url} to automatically generate your CLI token.')
                 if not webbrowser.open(self._login_url):
                     logging.debug("Failed to open the browser.")
-                    return None
+                    print(f'⚠️ Not able to open your browser automatically. Please go to {self._login_url} ⚠️')
+                    #return None
```

Comment:
Having emojis are cool, but I am a bit worried of how different computer's (old ones mostly) terminal would render those... not so sure, for now though this is not a problem

Reply:
I think it's fine if older computer have a weird experience for now I guess.